### ISSUE TITLE
ClusterOps - Set Up Kubernetes Cluster w/ Moondog

### ISSUE THREAD

**Issue by iamjoshfrank** [revelry-foundation/nightingale#17]

**Comment by CurtisVonRubenhoff** [comment]
I started on the flux config for the app. The pr is currently pointed at revelry-prod, but can be redirected when nightingale cluster is spun up

https://github.com/revelrylabs/our-flux/pull/345

**Comment by jwietelmann** [comment]
We are very close to finished here.

**Comment by jwietelmann** [comment]
@grossvogel is getting the YAML templates together to commit to https://github.com/revelry-foundation/nightingale and then we should theoretically be off to the races.

**Comment by grossvogel** [comment]
@CurtisVonRubenhoff You'll want to make your PR for app setup in the https://github.com/revelry-foundation/nightingale-cluster repo, against the `master` branch.

@jwietelmann The flux stuff is merged, but now I'm noticing this in the flux logs:
```
git repo not ready: attempt to push tag: fatal: Could not read from remote repository., full output:\n ERROR: Permission to revelry-foundation/nightingale-cluster.git denied to revelry-foundation-machine-user.\nfatal: Could not read from remote repository.\n\nPlease make sure you have the correct access rights\nand the repository exists.\n
```

It _seems_ like it's decoding the ssh key fine and mapping it to the right GitHub user, but running into a permission problem?

**Comment by jwietelmann** [comment]
I just gave `revelry-foundation-machine-user` read/write on the cluster repo.

**Comment by grossvogel** [comment]
Cool, thanks! Looks like it's working now (despite flux logs being _really_ hard to parse)

**Comment by CurtisVonRubenhoff** [comment]
@grossvogel @jwietelmann gonna get those files added to that repo. can one of your fine gentlemen help me with a sealed s3 secret to add in

**Comment by grossvogel** [comment]
Grafana OAuth isn't working properly and my best guess is that it's caused by some unescaped special characters in the client secret. (Grafana consumes its config via an `.ini` file.) Working on a PR to update that and I'll build on revelry-foundation/nightingale-cluster#3 to make sure I don't blow away @jwietelmann's work there

**Comment by grossvogel** [comment]
After a small hitch on the grafana client secret fix, I got it deployed and things are working.
- [X] Grafana with OAuth
- [X] Gangway OAuth to get kubectl creds
- [X] Prometheus showing happy metrics targets (with oauth proxy)
- [X] Flux is updating things
- [X] Velero logs look healthy and I see backups in S3
- [ ] Harbor is healthy but we haven't pushed or pulled any images AFAIK
- [ ] Navigator isn't in yet AFAIK


Cert manager, nginx, sealed secrets, and user permissions are visibly working based on the other things

**Comment by grossvogel** [comment]
We tested (and fixed) Harbor while we were working on getting the prod and staging servers deployed, and it's working now. We've decided to pass on navigator, for now anyway.

I believe this issue is done.

**Comment by iamjoshfrank** [comment]
I believe this is done. We're ... deploying somewhere, right? Closing.
